cdap-master/pom.xml

```diff
@@ -138,16 +138,10 @@
     <profile>
       <id>dist</id>
       <properties>
+        <etl.batch.plugin.name>etlBatch</etl.batch.plugin.name>
+        <etl.realtime.plugin.name>etlRealtime</etl.realtime.plugin.name>
```

Comment:
You can define a properties file in the src/main/resources/etl.properties have content as:

```
etl.batch.plugin.name=${etl.batch.plugin.name}
etl.realtime.plugin.name=${etl.realtime.plugin.name}
```

then in the Java profile (e.g. ETLReadltimeTemplate.java), you can get it by:

```java
Properties prop = new Properties();
InputStream input = getClass().getResourceAsStream("/etl.properties");
prop.load(input);
...
configurer.setName(prop.get("etl.realtime.plugin.name"));
...
```

This ensure the name is defined in once place.